src/code.sh

```
@@ -8,32 +8,33 @@ dx-download-all-inputs
 #Extract sample name from input_bam
-sample_name=$(echo $bam_prefix | cut -d '_' -f 1)
+sample_name=$(echo $bam_prefix | cut -d '.' -f 1)
 
 # Run arriba
-arriba \
-    -x $bam_path \
-    -o /home/dnanexus/out/output_fusions_files/${sample_name}_fusions.tsv -O /home/dnanexus/out/output_discarded_fusions/${sample_name}_fusions.discarded.tsv \
-    -a /home/dnanexus/genome_lib/${lib_dir}/ctat_genome_lib_build_dir/ref_genome.fa -g /home/dnanexus/genome_lib/${lib_dir}/ctat_genome_lib_build_dir/ref_annot.gtf \
-    -b /home/dnanexus/arriba/database/blacklist_hg38_GRCh38_v*.tsv.gz -k /home/dnanexus/arriba/database/known_fusions_hg38_GRCh38_v*.tsv.gz -t /home/dnanexus/arriba/database/known_fusions_hg38_GRCh38_v*.tsv.gz -p /home/dnanexus/arriba/database/protein_domains_hg38_GRCh38_v*.gff3
+
+docker_cmd="arriba_v*/arriba -x /data/in/bam/$bam_name \
```

Shellcheck (0.10.0)

[warning] bam_name is referenced but not assigned.

(SC2154)